javascript/ql/src/semmle/javascript/security/dataflow/HttpToFileAccess.qll

```diff
@@ -1,30 +1,33 @@
-/** 
- * Provides taint tracking configuration for reasoning about files created from untrusted http downloads.  
+/**
+ * Provides a taint tracking configuration for reasoning about user-controlled data in files.
```

Comment:
Again, I'd emphasize that it's *written* to files.

Otherwise it sounds like it's looking for files in the codebase that have "user-controlled data" (which doesn't really make sense).